vendor/bat-native-ads/include/bat/ads/mojom.h

```diff
@@ -6,13 +6,16 @@
 #ifndef BAT_ADS_MOJOM_H_
 #define BAT_ADS_MOJOM_H_
 
-#include "bat/ads/public/interfaces/ads_database.mojom.h"
-#include "bat/ads/public/interfaces/ads.mojom.h"
+#include "brave/vendor/bat-native-ads/include/bat/ads/public/interfaces/ads.mojom.h"
+#include "brave/vendor/bat-native-ads/include/bat/ads/public/interfaces/ads_database.mojom.h"

```

Comment:
Why did we add `brave/vendor/bat-native-ads-include/` as this should not be needed?